23-caching-with-javalin/COURSE_MATERIAL.md

```diff
@@ -0,0 +1,844 @@
+[markdown]:
+  https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/23-caching-with-javalin/COURSE_MATERIAL.md
+[pdf]:
+  https://heig-vd-dai-course.github.io/heig-vd-dai-course/23-caching-with-javalin/23-caching-with-javalin-practical-work.pdf
+[license]:
+  https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/LICENSE.md
+[discussions]: https://github.com/orgs/heig-vd-dai-course/discussions/122
+[illustration]:
+  https://images.unsplash.com/photo-1492515114975-b062d1a270ae?fit=crop&h=720
+
+# Caching with Javalin - Course material
```

Comment:
I don't understand why this course material is under the chapter "caching with javalin" while talking about web infrastructures ?